packages/diagrams/lib/src/grid_view.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+
+import 'diagram_step.dart';
+
+const String _gridView = 'grid_view';
+const String _customScrollGridView = 'grid_view_custom_scroll';
+
+class GridViewDiagram extends StatelessWidget implements DiagramMetadata {
+  const GridViewDiagram(this.name);
+
+  @override
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+    Widget returnWidget;
+
+    switch (name) {
+      case _gridView:
+        returnWidget = GridView.count(
+          primary: false,
+          padding: const EdgeInsets.all(20.0),
+          crossAxisSpacing: 10.0,
+          crossAxisCount: 2,
+          children: <Widget>[
+            Container(child: const Center(child: Text('He\'d have you all unravel at the')), color: Colors.teal[100]),
```

Comment:
Okay, apologies for the iteration. Maybe `Center` was not the best idea since some of these strings are longer than others, and the longer ones come right up to the top/left edges. How about instead:
```suggestion
            Container(child: const Padding(padding: EdgeInsets.all(8.0), child: Text('He\'d have you all unravel at the')), color: Colors.teal[100]),
```
How does that look?